core/string/locales.h

```diff
@@ -1057,8 +1057,8 @@ static const char *script_list[][2] = {
 	{ "Hangul", "Hang" },
 	{ "Han", "Hani" },
 	{ "Hanunoo", "Hano" },
-	{ "Simplified", "Hans" },
-	{ "Traditional", "Hant" },
+	{ "Han (Simplified variant)", "Hans" },
```

Comment:
Dosen't this have the same issue that you said about double parentheses?

Reply:
Since we already have such names and this is a standard name, I think it might be acceptable.

Reply:
The only other case is `Myanmar (Burmese)`, maybe it's better to change it to `Myanmar / Burmese` as well and use `Simplified Han` + `Traditional Han` to avoid extra parentheses.